src/impact-dashboard/FacilityInformation.tsx

```diff
@@ -80,46 +76,86 @@ export const AgeGroupGrid: React.FC<AgeGroupGridProps> = ({
   ...props
 }) => {
   const [collapsed, setCollapsed] = useState(collapsible);
+
+  const passedAgesKnown = (model: Record<string, any> | undefined) => {
+    if (model !== undefined) {
+      let keys = Object.keys(model);
+      return keys.some(function (key) {
+        return RegExp(/age\d+/).test(key);
+      });
+    } else {
+      return true;
+    }
+  };
+
+  const passedAgesUnknown = (model: Record<string, any> | undefined) => {
+    if (model !== undefined) {
+      let keys = Object.keys(model);
+      return keys.some(function (key) {
+        return RegExp(/ageUnknown\w+/).test(key);
+      });
+    } else {
+      return true;
+    }
+  };
+
+  const collapseAgeInputs = (model: Record<string, any> | undefined) => {
+    if (passedAgesKnown(model)) {
+      return false;
+    } else if (
+      passedAgesKnown(model) === false &&
+      passedAgesUnknown(model) === true
```

Comment:
I don't know whether it's in scope but I think we might consider having this be a form validation condition as well as a UI display rule? I don't see anything here that actually prevents you from entering stuff in both fields, although we are certainly making it harder to do so

Reply:
I'd be happy to add some form validation, although I'm not sure what you have in mind.

I see the value of keeping this PR smaller/simpler and I could do form validation in the next round. Maybe we should write up an issue on form validation? If you you me more guidance I'll try to write it up.

Reply:
yeah that sounds fine. the form validation, as I imagine it, would be pretty similar to what you've already done. e.g. valid would be `passedAgesUnknown (xor) passedAgesKnown` ... invalid would be `passedAgesUnknown && passedAgesKnown`. that way we try to actually enforce mutual exclusivity